dist/practice.js

```diff
@@ -4295,16 +4311,17 @@ var SIM_CONTROLLER = (function () {
 				// Check Nullify
 				if (target.nullified && !skill.ignore_nullify) {
 					target.nullified--;
-					if (debug) echo += debug_name(src_card) + ' enhances ' + debug_name(target) + ' but it is nullified!<br>';
+					if (debug) echo += debug_name(src_card) + ' imbues ' + debug_name(target) + ' but it is nullified!<br>';
```

Comment:
💚 Another good catch.

There's enough logging fixes here to be their own PR at this point, but I'm fine with leaving them in this one, now that they're here.